.rubocop.yml

```diff
@@ -876,6 +872,8 @@ Style/UnneededPercentQ:
   Description: Checks for %q/%Q when single quotes or double quotes would do.
   StyleGuide: https://github.com/bbatsov/ruby-style-guide#percent-q
   Enabled: true
+  Exclude:
+    - tmuxinator.gemspec
```

Comment:
Is the gemspec file "special" enough that we should just exclude it under `AllCops`, instead of a handful of individual cops?